AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/backgroundfilter/BackgroundFilterProcessor.swift

```diff
@@ -221,27 +204,50 @@ public class BackgroundFilterProcessor {
         return bufferPool
     }
 
-    /// Resize an image using `CILanczosScaleTransform` CIFilter.
+    /// Resize CGImage to the given size.
     ///
     /// - Parameters:
-    ///   - image: Image to scale.
-    ///   - scale: Scaling factor.
-    ///   - aspectRatio: Aspect ratio factor.
-    func resizeImage(image: CIImage, scale: Double, aspectRatio: Double) -> CIImage? {
-        guard let resizeFilter = CIFilter(name: "CILanczosScaleTransform") else {
-            logger.error(msg: "Error creating CILanczosScaleTransform CIFilter.")
+    ///   - image: Input image to resize.
+    ///   - newSize: Size of the image output.
+    ///
+    /// - Returns: Resized image.
+    private func resizeImage(image: CGImage, newSize: CGSize) -> CGImage? {
+        guard let colorSpace = image.colorSpace else {
```

Comment:
CIFilter is supposed to be faster since it can utilize GPU. I didn't notice significantly any difference between two performance wise though (FPS and utilization was the same when testing).